--- nagios-plugin-check_poldek-0.6/check_poldek.py~	2012-01-11 22:45:49.000000000 +0200
+++ nagios-plugin-check_poldek-0.6/check_poldek.py	2012-01-11 22:53:57.124944772 +0200
@@ -34,8 +34,8 @@
     "errorLevel": 10,
     "warningLevel": 5,
     "verbose": False,
-    "sources": [],
-    "cache": "/tmp/check_poldek",
+    "sources": @repos@,
+    "cache": "/var/cache/check_poldek",
     "extraArgs": [],
 }
 
